File: Javascript/Algorithms/1876.js
/**
 * @param {string} s
 * @return {number}
 */
var countGoodSubstrings = function(s) {
  let count = 0, i = 2, end = s.length;
  while (i < end) {
    let a = s[i - 2], b = s[i - 1], c = s[i];
    if (a == b && b == c) {
      i += 2;
      continue;
    }
    count += a != b && b != c && a != c;
    i++;
  }
  return count;
};

File: C/Algorithms/1876.c
int countGoodSubstrings(char *s) {
  // yes, could do it w/o strlen but meh.
  int count = 0, i = 2, end = strlen(s);
  while (i < end) {
    char a = s[i - 2], b = s[i - 1], c = s[i];
    if (a == b && b == c) {
      i += 2;
      continue;
    }
    count += a != b && b != c && a != c ? 1 : 0;
    i++;
  }
  return count;
}